### DISCUSSION TITLE
Loihi 2 Microcode Documentation

### DISCUSSION THREAD

**Discussion by kds300** [lava-nc/lava#268] [General]
If I recall correctly, one of the sessions of the Spring 2022 workshop previewed some of the microcode that will be used to program the neurocores on the Loihi 2 chip (I can't find which recording it was in). Is there any documentation available for the microcode? This would be very helpful in preparing to run networks once the newest version of Lava is released and to make sure that the processes we write CPU models for are achievable on the Loihi 2 chip.

**Comment by awintel** [comment]
The upcoming release won't yet have comprehensive documentation for uCode programming but two first examples.
Will provide more in-depth documentation in the next release.
However, to access this feature, you must first become an INRC member.

**Reply by kds300** [reply]
Thanks! Are these examples currently available anywhere for members? I am currently an INRC member, I'm working on the speech recognition project with Dr. Jin.

**Reply by awintel** [reply]
No it's not available to anyone yet. We are waiting for one more internal approval which we should get tonight. So the release should finally go out tomorrow.
